docs/work/coverage-notes/README.md

```diff
@@ -0,0 +1,25 @@
+ 
+ # Introduction
+ When done correctly, coverage notes allow you to take the time you need off without resulting in any headaches for you or the team. In your coverage notes you should provide 3 things: What you are currently working on, what the status of your work is, and any other important notes that might be helpful to someone picking up the task. In conjunction with coverage notes, you should also talk to you PM(s) about anything else you might have missed. Your coverage notes should then be sent to your PM or most senior Dev.
+
+# Template
+
+So long as you provide notes for the 3 peices above, you are free to format the notes however you'd like. Below is an example ... 
+
+## Example 
+
+## Client #1
```

Comment:
I think I said Client 1,2 when we talked about this but now that I'm reading it I'm not sure. How do you feel about it being "Client Name"? I'm a 2/5 on this so if you like it better the way it is I'm good with that.